h3.go

```diff
@@ -98,8 +98,10 @@ type (
 	}
 
 	// LinkedGeoPolygon is a linked-list of GeoPolygons.
-	// TODO: not implemented.
-	LinkedGeoPolygon struct{}
+	LinkedGeoPolygon struct {
```

Comment:
In uber/h3, this seems to be doubly-linked: https://github.com/uber/h3/blob/138a7487adb0892715ea7ced785db4e9c38842d1/src/h3lib/include/h3api.h.in#L179

Does that need to be maintained here?